### ISSUE TITLE
Sign in menu requires that sub-items be children

### ISSUE THREAD

**Issue by allisonmuise** [Esri/crowdsource-reporter#382]
Parent:
![11-14-2018 1-52-42 pm](https://user-images.githubusercontent.com/2752195/48515020-9a44f300-e814-11e8-95f7-cdd43aa8a63a.png)
![11-14-2018 1-52-32 pm](https://user-images.githubusercontent.com/2752195/48515021-9a44f300-e814-11e8-8968-3fa91fa8bc98.png)

Child 1:
![11-14-2018 1-53-52 pm](https://user-images.githubusercontent.com/2752195/48515161-ed1eaa80-e814-11e8-9785-2f4c06131d3a.png)
![11-14-2018 1-54-00 pm](https://user-images.githubusercontent.com/2752195/48515162-ed1eaa80-e814-11e8-9089-a3e4ddd0a688.png)

Child 2:
![11-14-2018 1-54-31 pm](https://user-images.githubusercontent.com/2752195/48515173-f27bf500-e814-11e8-9772-62085b528c99.png)
![11-14-2018 1-54-23 pm](https://user-images.githubusercontent.com/2752195/48515175-f27bf500-e814-11e8-89ec-a2128fded688.png)

**Comment by ashishchoure** [comment]
@allisonmuise 

To allow user to easily access the menu in desktop and mobile devices, we have added a role as `"menu"` and `"menuitem"`. Although the sign-in menu looks like a menu, its basically constructed using DIV's. The parent is created in application header and the menu items are just floating somewhere else. This is the reason we are seeing an issue in accessibility tool.

The standards says that, menu should be created using list control. This will be a considerable change in design and we may need to redesign the menu control's as we need to do this for mobile and desktop mode as well.

We also came across this issue and we added this in delivery mail to keep you updated about the list of issues that we are seeing.
